documents/overview.md

```diff
@@ -0,0 +1,42 @@
+<!--suppress HtmlDeprecatedAttribute -->
```

Comment:
Please move this file to the main README (and remove the current OID4VP demo section with our ugly UI from it). The current demo is mainly about VCI, but also can use VP, so it covers that use case as well.

The reason this needs to move to the main README is to make sure people will actually find this documentation on how to setup everything

Reply:
moved and merged original readme with this document